### DISCUSSION TITLE
Support for codesigning and notarization

### DISCUSSION THREAD

**Discussion by achou11** [neutralinojs/neutralinojs#761] [Q&A] [answered]
Does Neutralino support codesigning (and notarization, for OSX) out of the box or is that treated as an independent process left for the end-user to implement?

**Comment by shalithasuranga** [comment, marked as answer]
Hey.. Currently, Neutralinojs CLI provides binaries for all supported platforms. But it doesn't support customizing icons, building installers, and code signing, etc since we focus mainly on developing the framework. We hope the developer community will build those packages someday like Electron community-built [packages](https://github.com/electron-userland). For example, you can see Electron's code signing implementation is a [part of `electron-builder`](https://github.com/electron-userland/electron-builder/blob/87300278d24e8304caa4b053b883843a2447dab2/packages/app-builder-lib/src/codeSign/macCodeSign.ts).  FYI, our developer community built an [app image builder](https://www.npmjs.com/package/neutralino-appimage-bundler) for Neutralinojs. 

We introduced a [plugin system](https://github.com/neutralinojs/neutralinojs-cli#plugins) for the CLI too to support these community packages.

Thanks for discussing this :)

**Reply by achou11** [reply]
makes sense! hoping that support will come one day via the community (maybe I'll have to take it into my own hands 😅)

tangentially, is there a listing of helpful community projects related to Neutralino (something like an Awesome Neutralino repo)? Would be great to consolidate resources to make such projects more discoverable and grow the ecosystem :)

**Reply by shalithasuranga** [reply]
Hey.. That's a great idea. Thanks. I usually enter "neutralinojs" on GitHub and see latest results frequently. There are app templates, apps, web-based apps, etc. Do you think awesome-repo or a page in docs is more effective? seems like awesome-repo? :D

**Reply by shalithasuranga** [reply]
Like this https://github.com/sindresorhus/awesome-electron ? maybe ;)

**Reply by shalithasuranga** [reply]
If you are interested in creating the repo please create it - since you've suggested the idea. I will also add things I've found on GitHub

**Reply by achou11** [reply]
yeah definitely would lean towards creating a repo similar to what Electron has!

think it would make more sense for it to be under the Neutralino organization but happy to create it under mine if that's preferred :)

**Reply by shalithasuranga** [reply]
Great :) .. mm I think you can create the repo may be in your profile or (your org -  as you wish) since you've suggested the idea and the repo is more related to community support. I'll add a link to your repo from the main repo ;) And I think https://www.reddit.com/r/coolgithubprojects/ is a nice place to let others know about it :)

**Reply by achou11** [reply]
Created the repo (https://github.com/achou11/awesome-neutralino) and added you as a collaborator :)

**Reply by shalithasuranga** [reply]
Hey.. Thank you very much.. I'll send you a pull request with some updates soon :)
